libs/crypt.js

```diff
@@ -0,0 +1,33 @@
+const CryptoJS = require('crypto-js');
+
+const characters = '0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ';
+
+function randomString(length) {
```

Comment:
should we use this or should we make use of CryptoJS.lib.WordArray like they do in their AES examples on https://cryptojs.gitbook.io/docs/#the-cipher-input

Reply:
No. We should pass a string which is then derived to its key and IV, See the docs you linked:

> For the key, when you pass a string, it's treated as a passphrase and used to derive an actual key and IV.

Reply:
They use it in this example to generate the salt https://cryptojs.gitbook.io/docs/#pbkdf2 which is why I noticed it

Reply:
it looks like v4 added this to avoid a security issue CryptoJS.lib.WordArray it isn't clear from the release notes but that's why they do not use math.random anymore https://github.com/brix/crypto-js#400

Reply:
We use version 4 and the `Math.random` has been patched by their method, see:

> In this version Math.random() has been replaced by the random methods of the native crypto module.